spec/mongo/socket/ssl_spec.rb

```diff
@@ -50,6 +50,19 @@
   end
 
   describe '#connect!' do
+    context 'when TLS context hooks are provided' do
+      let(:proc) { Proc.new { |context| context.ciphers = ["AES256-SHA"] } }
+
+      before do
+        Mongo::TLSContextHooks.hooks = [ proc ]
```

Comment:
I suggest `Mongo.tls_context_hooks = [ proc ]` as the API, either delegated to Mongo::TlsContextHooks or defined on Mongo module itself.